### PR TITLE
Post new comment

### PR DESCRIPTION
**Pull request by theRohirrim**

### CODE REVIEW

File: src/pages/comments/CommentForm.jsx [hunk 1]
```diff
@@ -1,24 +1,80 @@
...
         setInput(event.target.value);
     };
 
+    const handleCollapsible = (event) => {
+        event.preventDefault()
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        console.log('enters submit')
```
Comment (saimakhanom) [inline src/pages/comments/CommentForm.jsx]: remove logs

File: src/pages/comments/CommentForm.jsx [hunk 2]
```diff
@@ -1,24 +1,80 @@
...
+        // Build comment
+        const comment = {
+            article_id: article_id,
+            author: currentUser,
+            body: input,
+            created_at: 'Now',
+            votes: 0
+        }
+
+        console.log('built comment', comment)
```
Comment (saimakhanom) [inline src/pages/comments/CommentForm.jsx]: and this one

File: src/pages/comments/CommentForm.jsx [hunk 3]
```diff
@@ -1,24 +1,80 @@
...
+
+        // Render optimistically
+        setComments((currentComments) => {
+            console.log([comment, ...currentComments])
```
Comment (saimakhanom) [inline src/pages/comments/CommentForm.jsx]: another :)

**Comment by saimakhanom** [comment]
We need to add some form validation to prevent the user from being able to post empty comments (and anything else you deem invalid)
Also the created at should be in amore human readable format so it makes sense to the user

**Comment by nc-hev-mag** [comment]
See comments on your other PR - thus is looking good to merge based on suggested changes - just consider the error handling tips I mentioned in the other PR :D

**Pull request merged**